source/_components/group.markdown

```diff
@@ -13,7 +13,7 @@ ha_category: Organization
 
 Groups allow the user to combine multiple entities into one. A group can be promoted to a **view** by setting the `view` option to `yes`. This will make the group available as a new tab in the frontend.
 
-Check the **Set State** page from the **Developer Tools** <img src='/images/screenshots/developer-tool-services-icon.png' class='no-shadow' height='38' /> and browse the **Current entities:** listing for all available entities.
+Check the **Set State** page from the **Developer Tools** ![<>](https://cdn.rawgit.com/Templarian/MaterialDesign/master/icons/svg/code-tags.svg) and browse the **Current entities:** listing for all available entities.
```

Comment:
You can't just link to external images, this all wrong. You have to link to the other developer tool image icon that's in the same dir.